### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'duo_api_key'

### ISSUE THREAD

**Issue by resilience-jychp** [lyft/cartography#1182]
*Description*:
> Error when launching cartography without configuring (or disabling) duo intel module.
>
> AttributeError: 'Namespace' object has no attribute 'duo_api_key'

*To Reproduce*:
> Launch cartography without duo config parameter and without specific module 

*Logs*:
```INFO:cartography.sync:Starting sync stage 'duo'
ERROR:cartography.sync:Unhandled exception during sync stage 'duo'
Traceback (most recent call last):
  File "/home/jeremy/Repos/cartography/cartography/sync.py", line 111, in run
    stage_func(neo4j_session, config)
  File "/home/jeremy/Repos/cartography/cartography/util.py", line 165, in timed
    return method(*args, **kwargs)
  File "/home/jeremy/Repos/cartography/cartography/intel/duo/__init__.py", line 58, in start_duo_ingestion
    config.duo_api_key,
AttributeError: 'Namespace' object has no attribute 'duo_api_key'
Traceback (most recent call last):
  File "/home/jeremy/.local/share/virtualenvs/cartography-ra6kuGgL/bin/cartography", line 33, in <module>
    sys.exit(load_entry_point('cartography', 'console_scripts', 'cartography')())
  File "/home/jeremy/Repos/cartography/cartography/cli.py", line 705, in main
    sys.exit(CLI(prog='cartography').main(argv))
  File "/home/jeremy/Repos/cartography/cartography/cli.py", line 686, in main
    return cartography.sync.run_with_config(self.sync, config)
  File "/home/jeremy/Repos/cartography/cartography/sync.py", line 189, in run_with_config
    return sync.run(neo4j_driver, config)
  File "/home/jeremy/Repos/cartography/cartography/sync.py", line 111, in run
    stage_func(neo4j_session, config)
  File "/home/jeremy/Repos/cartography/cartography/util.py", line 165, in timed
    return method(*args, **kwargs)
  File "/home/jeremy/Repos/cartography/cartography/intel/duo/__init__.py", line 58, in start_duo_ingestion
    config.duo_api_key,
AttributeError: 'Namespace' object has no attribute 'duo_api_key'
```

*Please complete the following information:*:
>  - Up to date cartography (commit: [0c852d26ec32e7dedda28d260290023ea98f3b7e](https://github.com/lyft/cartography/commit/0c852d26ec32e7dedda28d260290023ea98f3b7e))
>  - Python 3.8.16
>  - OS: Ubuntu 22.04

**Comment by Mehyar-ALS** [comment]
Hi, is it possible to disable the duo module? If yes, do you know how? Thanks.

**Comment by resilience-jychp** [comment]
@Mehyar-ALS you can use `selected-modules` argument to run only a sub set of modules.
It will solve your issue and greatly increase execution speed.

Ex: `--selected-modules gsuite,github`

**Comment by Mehyar-ALS** [comment]
That was a very quick replay! Thanks

**Comment by achantavy** [comment]
Marking as fixed by #1190, thanks @resilience-jychp !
